tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs

```diff
@@ -429,6 +460,7 @@ private void TryClearContext(HttpContext context)
             try
             {
                 context.Items.Remove(_httpContextScopeKey);
+                context.Items.Remove(_httpContextScopeKey + ".proxy");
```

Comment:
See my other comments about using a field for `_httpContextScopeKey + ".proxy"`.